bitarray/_bitarray.c

```diff
@@ -131,6 +222,8 @@ resize(bitarrayobject *self, idx_t nbits)
 {
     Py_ssize_t newsize;
     size_t _new_size;       /* for allocation */
+//    void *newmem;
```

Comment:
there are several commented out code snippets -- can those be cleaned out, or some documentation be added why they should be left in?